frontend/test/acceptance/pages/EnterDetails.scala

```diff
@@ -25,6 +25,10 @@ case class EnterDetails(val testUser: TestUser) extends Page with Browser {
 
   def fillInCardDeclinedProcessError() { CreditCard.fillInCardDeclinedProcessError() }
 
+  def changeCountry() { DeliveryAddress.selectUSA() }
```

Comment:
`changeCountry()` is a bit vague here, I think you're better off making this `changeCountryTo("US")` - and then you could change country a couple of times, to checking currency each time.